crates/primitives/block/src/lib.rs

```diff
@@ -13,10 +13,35 @@ use mp_felt::Felt252Wrapper;
 use mp_hashers::HasherT;
 use mp_transactions::compute_hash::ComputeTransactionHash;
 use mp_transactions::Transaction;
+use serde::{Deserialize, Serialize};
```

Comment:
Create a `serde` feature, and derive `serde` in the same way `parity-scale-codec` is

Reply:
Done

Reply:
Actually I've removed optional feature for serde in Block as it is always needed for rename.

Reply:
I don't think that it is how it works.

```rust
#[derive(Debug, Clone, Copy, PartialEq, Eq, Deserialize, Serialize)]
#[cfg_attr(feature = "serde", derive(Deserialize, Serialize))]
pub enum BlockTag {
    #[cfg_attr(feature = "serde", serde(rename = "latest"))]
    Latest,
    #[cfg_attr(feature = "serde", serde(rename = "pending"))]
    Pending,
}
```

And "serde" should still be optional, and should be behind a feature flag